tests/keyframes.rs

```diff
@@ -0,0 +1,20 @@
+#![cfg(test)]
+
+#[macro_use]
+mod macros;
+
+test!(
+    keyframes_basic_toplevel,
+    "@keyframes foo {\n  to {\n    a {\n      color: red;\n    }\n  }\n}\n"
```

Comment:
this would error with the message `Error: Expected "to" or "from".` because it contains `a` as a selector

Reply:
Oh, I didn't know one cannot put `a` inside.